src/ScheduleManager.cpp

```diff
@@ -754,4 +758,4 @@ void ScheduleManager::ClearThreadTasklets()
 unsigned long ScheduleManager::ThreadId() const
 {
 	return m_threadId;
-}
\ No newline at end of file
+}
```

Comment:
Nitpick :D

Reply:
Interesting... generally, newline at end of file should be enforced by our `.editorconfig` - but I just realised that the `.editorconfig` in our carbon-template repository is dangerously wrong (and way too convoluted!), e.g. it wants to enforce windows-style line endings (`\r\n`) while our platform agnostic coding style guide recommends to always use `\n`. Guess _that_ needs to get fixed up, so I'll just leave this here :D